Don't prefix identifier with `url:`

brendanny opened the pull request:
Addresses https://github.com/standardebooks/tools/issues/831

apasel422 commented:
Looks like https://github.com/standardebooks/tools/blob/0cf06403b2d0722687759500a919fed2ab959342/tests/file_commands/extract-ebook/test-1/golden/jane-austen_unknown-novel.epub.extracted/epub/content.opf#L4 also needs to be updated.

brendanny commented:
Looks fine on my end

https://github.com/standardebooks/tools/pull/834/files#diff-cb578716173f35be8ace6a64d6f62d70e03e78afa2964da879fb3cf82c9efcfd

acabal commented:
The tests are failing however, see the checks section at the bottom of this PR.

brendanny commented:
I can't make too much sense of the test output, but I think they are failing because the github action is using the most recent tools release which has the `url:` prefix in the golden files.

brendanny commented:
Oh wait, no. We have full epubs in here that I need to fix

brendanny commented:
which one has the correct css?

- [tests/build/test-1/golden/extract/epub/css/se.css](https://github.com/standardebooks/tools/blob/master/tests/build/test-1/golden/extract/epub/css/se.css)
- [tests/file_commands/extract-ebook/test-1/golden/jane-austen_unknown-novel.epub.extracted/epub/css/se.css](https://github.com/standardebooks/tools/blob/master/tests/file_commands/extract-ebook/test-1/golden/jane-austen_unknown-novel.epub.extracted/epub/css/se.css)

```
@@ -66,6 +66,11 @@
        width: 25%;
 }
 
+section.epub-type-colophon p.nth-last-child-2 time,
+section.epub-type-colophon p:nth-last-child(2) time{
+       font-variant: small-caps;
+}
+
 section.epub-type-colophon a{
        font-variant: small-caps;
 }
```

acabal commented:
The one in `tests/build/...`

brendanny commented:
these epub test will be the death of me... A full round-trip test seems more maintainable (no binary files in-repo, no tracking separate golden files for each, etc...)

One thing to note, and why my latest build failed :(
- `se clean` changes the css in the extract and build tests.

acabal commented:
I suggest just copying the tests from master and pasting them into your PR. Tests are passing in master.

brendanny commented:
Okay, this should be final and ready to merge now.

acabal commented:
Great work, thanks!
